github/compliance/report.py

```diff
@@ -294,21 +314,32 @@ def process_result(
             )
         elif action == 'report':
             if delivery_svc_client:
-                assignees = delivery.client.github_users_from_responsibles(
-                    responsibles=delivery_svc_client.component_responsibles(
-                        component=component,
-                        artifact=artifact,
-                    ),
-                    github_url=repository.url,
-                )
-
-                assignees = tuple((
-                    u.username for u in assignees
-                    if github.user.is_user_active(
-                        username=u.username,
-                        github=gh_api,
+                try:
+                    assignees = delivery.client.github_users_from_responsibles(
+                        responsibles=delivery_svc_client.component_responsibles(
+                            component=component,
+                            artifact=artifact,
+                        ),
+                        github_url=repository.url,
                     )
-                ))
+
+                    assignees = tuple((
+                        u.username for u in assignees
+                        if github.user.is_user_active(
+                            username=u.username,
+                            github=gh_api,
+                        )
+                    ))
+                except requests.exceptions.HTTPError as e:
+                    if e.response.status_code == 404:
+                        logger.warning(f'Delivery Service returned 404 for {component.name=}, '
+                            f'{artifact.name=}')
+                        assignees = ()
+                        target_milestone = None
+                    else:
+                        raise
+                else:
+                    raise
```

Comment:
I think this has the wrong indentation.
try...else is valid in python. This will be executed if no exception is raised in the try-block

Reply:
Good catch... Thanks